plugins/janus_streaming.c

```diff
@@ -3161,7 +3165,7 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 					doaudio, (audiopt ? json_integer_value(audiopt) : -1),
 						(char *)json_string_value(audiortpmap), (char *)json_string_value(audiofmtp),
 					dovideo, (videopt ? json_integer_value(videopt) : -1),
-						(char *)json_string_value(videortpmap), (char *)json_string_value(videofmtp),
+						(char *)json_string_value(videortpmap), (char *)json_string_value(videofmtp), videobufferkf ? json_is_true(videobufferkf) : FALSE,
```

Comment:
Just for code style, please move the new part to a new line (too long as it is).

Reply:
Fixed. I kept an extra indentation level to align with other video params, I hope it's ok.